tests/test_venue_request.py

```diff
@@ -398,7 +398,8 @@ def test_venue_revision(self, client, test_client, selenium, request_page, venue
                     'preprint': {
                         'value-regex': '.*'
                     }
-                }
+                },
+                'reviewer_roles': ['Reviewers', 'Expert Reviewers']
```

Comment:
I know this is a test but the role should be Expert_Reviewers (like we have Area_Chairs or Senior_Area_Chairs). Having Expert Reviewers will try to create the group `TEST.cc/2030/Conference/Expert Reviewers` and that will give an error.